service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/JudgeAndLegalAdvisor.java

```diff
@@ -35,4 +35,11 @@ public static JudgeAndLegalAdvisor from(final Judge allocatedJudge) {
         }
         return judgeAndLegalAdvisorBuilder.build();
     }
+
+    public JudgeAndLegalAdvisor resetJudgeProperties() {
```

Comment:
why not static?

Reply:
If static I cannot reference legalAdvisorName and would have to pass it as parameter instead. Which do you prefer? I think I will keep as non-static but i'm curious